helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java

```diff
@@ -68,32 +59,25 @@
 import static com.google.common.base.Optional.fromNullable;
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
-import static com.google.common.base.Strings.isNullOrEmpty;
 import static com.google.common.collect.Lists.newArrayList;
-import static com.spotify.helios.testing.Jobs.undeploy;
 import static java.lang.String.format;
-import static java.lang.System.getProperty;
-import static java.util.Arrays.asList;
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class TemporaryJobs implements TestRule {
```

Comment:
This would make it very simple to add TestNG support too, and simplify the case where people want to use TemporaryJobs in something that isn't a test.